spec/tanker/identity_spec.rb

```diff
@@ -167,6 +169,29 @@
       expect(identity['public_signature_key']).to eq('W7QEQBu9FXcXIpOgq62tPwBiyFAbpT1rAruD0h/NrTA=')
       expect(identity['public_encryption_key']).to eq('/2j4dI3r8PlvCN3uW4HhA5wBtMKOcACd38K6N0q+mFU=')
     end
+
+    it 'a valid sms provisional identity' do
```

Comment:
misses a verb?

Reply:
It's because we're in "describe('parse')" I think, but sure, added the verbs